gensrc/script/vectorized/vectorized_functions.py

```diff
@@ -351,6 +351,7 @@
     [50340, 'date_trunc', 'DATETIME', ['VARCHAR', 'DATETIME'], 'TimeFunctions::datetime_trunc', 'TimeFunctions::datetime_trunc_prepare', 'TimeFunctions::datetime_trunc_close'],
     [50350, 'date_trunc', 'DATE', ['VARCHAR', 'DATE'], 'TimeFunctions::date_trunc', 'TimeFunctions::date_trunc_prepare', 'TimeFunctions::date_trunc_close'],
     [50360, 'timestamp', 'DATETIME', ['DATETIME'], 'TimeFunctions::timestamp'],
+    [50370, 'date_floor', 'DATETIME', ['DATETIME', 'INT', 'VARCHAR'], 'TimeFunctions::datetime_floor', 'TimeFunctions::datetime_floor_prepare', 'TimeFunctions::datetime_floor_close'],
```

Comment:
Do these function have limits?
1. For example, does this function support date_floor(date_column, interval 90 seconds)?
2. Does the second argument support variable? such as date_floow(date_column , interval second_column seconds)?

Reply:
> Do these function have limits?
> 
>     1. For example, does this function support date_floor(date_column, interval 90 seconds)?
> 
>     2. Does the second argument support variable? such as date_floow(date_column , interval second_column seconds)?


1. yes,`date_floor` support any positive interval
2. no, `date_floor` requires second parameter must be a constant interval

Above limits have been verified in FE. @imay